packages/server/src/api/routes/tests/table.spec.ts

```diff
@@ -182,6 +181,49 @@ describe("/tables", () => {
         1
       )
     })
+
+    it("should update Auto ID field after bulk import", async () => {
+      const table = await config.createTable({
+        name: "TestTable",
+        type: "table",
+        schema: {
+          autoId: {
+            name: "id",
+            type: FieldType.NUMBER,
+            subtype: "autoID",
+            autocolumn: true,
+            constraints: {
+              type: "number",
+              presence: false,
+            },
+          },
+        },
+      })
+
+      let response = await request
```

Comment:
These tests are great - the only one small improvement we could make is to use the new test APIs to make it a bit more verbose/easier to read, these can be accessed through the `config.api` object, for example for the row creation below we could do:
```
const tableId = table._id!
let response = await config.api.row.save(tableId, {})
expect(response.body.autoId).toEqual(1)
await config.api.table.bulkImport(tableId, [{ autoId: 2 }])
response = await config.api.row.save(tableId, {})
expect(response.body.autoId).toEqual(3)
```

The only thing we would need to do here is add a `bulkImport` to the table API file, which is pretty much just your bulk API request below!